### PR TITLE
LOC CHECKIN | dotnet/roslyn dev15.7.x-vs-deps | 20180409

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by tmeschter** [comment]
@v-zbsail Would you be able to fix up these merge conflicts?

**Comment by v-zbsail** [comment]
@tmeschter Not really. I'm sorry, but I'm only providing the the localized content on behalf of Moravia production. Any conflicts should be resolved on your side.

**Comment by tmeschter** [comment]
@jaredpar The Spanish leg is failing on a compiler test.

**Comment by AlekseyTs** [comment]
Microsoft.CodeAnalysis.BuildTasks.UnitTests.MapSourceRootsTests.SourceRootPaths_EndWithSeparator (from Microsoft.Build.Tasks.CodeAnalysis.UnitTests)
```
Expected:
ERROR : SourceRoot paths are required to end with a slash or backslash: 'C:'
ERROR : SourceRoot paths are required to end with a slash or backslash: 'C'
Actual:
ERROR : Se requiere que las rutas SourceRoot finalicen con una barra diagonal o una barra diagonal inversa: \"\"C:\"\"
ERROR : Se requiere que las rutas SourceRoot finalicen con una barra diagonal o una barra diagonal inversa: \"\"C\"\"
Differences:
++> ERROR : Se requiere que las rutas SourceRoot finalicen con una barra diagonal o una barra diagonal inversa: \"\"C:\"\"
++> ERROR : Se requiere que las rutas SourceRoot finalicen con una barra diagonal o una barra diagonal inversa: \"\"C\"\"
--> ERROR : SourceRoot paths are required to end with a slash or backslash: 'C:'
--> ERROR : SourceRoot paths are required to end with a slash or backslash: 'C'

Expected: True
Actual: False
```

**Comment by AlekseyTs** [comment]
It looks like the test was added recently by @tmat. I would recommend to adjust the test to request diagnostics in english

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf [hunk 1]
```diff
@@ -2698,7 +2698,7 @@ Pokud se taková třída používá jako základní třída a pokud odvozující
       </trans-unit>
       <trans-unit id="ERR_MissingStructOffset">
         <source>'{0}': instance field in types marked with StructLayout(LayoutKind.Explicit) must have a FieldOffset attribute</source>
-        <target state="needs-review-translation">'{0}: Typy polí instance označené deklarací StructLayout(LayoutKind.Explicit) musí mít atribut FieldOffset.</target>
+        <target state="translated">{0}: Typy polí instance označené deklarací StructLayout(LayoutKind.Explicit) musí mít atribut FieldOffset.</target>
```
Comment (Neme12) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf]: This translation is wrong.

Comment (Neme12) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf]: It says the type of the field should have the attribute, which is incorrect.

**Comment by tmat** [comment]
looking

**Comment by tmat** [comment]
Fixed here: https://github.com/dotnet/roslyn/pull/26134

**Pull request merged**